package.json

```diff
@@ -9,7 +9,7 @@
     "node": ">=4"
   },
   "scripts": {
-    "test": "xo && ava"
+    "test": "FORCE_HYPERLINK=1 xo && ava"
```

Comment:
This is to make sure the `hyperlinks` test also works in terminals which doesn't support hyperlinks.

This made me wonder, should AVA have a `env` setting in `package.json` like this? It just popped up when I had to do this.

```js
{
    "ava": {
        "env": {
            "FORCE_HYPERLINK": 1
        }
    }
}
```